docs/qcqmc/full_workflow.ipynb

```diff
@@ -0,0 +1,363 @@
+    "\n",
+    "blueprint_params = BlueprintParamsTrialWf(\n",
+    "    name=\"blueprint_test_medium\",\n",
+    "    trial_wf_params=pp_params,\n",
+    "    n_cliffords=100,\n",
+    "    qubit_partition=(tuple(qubit for qubit in pp_params.qubits_jordan_wigner_ordered),),\n",
+    "    seed=1,\n",
+    ")\n",
+    "\n",
+    "blueprint = BlueprintData.build_blueprint_from_dependencies(blueprint_params, dependencies={pp_params: trial_wf})\n",
+    "\n",
+    "simulated_experiment_params = SimulatedExperimentParams(\n",
+    "    name=\"test_1\",\n",
+    "    blueprint_params=blueprint.params,\n",
+    "    noise_model_name=\"None\",\n",
+    "    noise_model_params=(0,),\n",
+    "    n_samples_per_clifford=31,\n",
+    "    seed=1,\n",
+    ")\n",
+    "experiment = ExperimentData.build_experiment_from_dependencies(\n",
+    "    params=simulated_experiment_params, dependencies={blueprint.params: blueprint}\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import Dict\n",
+    "from recirq.qcqmc.data import Params, Data\n",
+    "from recirq.qcqmc.analysis import OverlapAnalysisData, OverlapAnalysisParams\n",
+    "\n",
+    "analysis_params = OverlapAnalysisParams(\n",
+    "    \"TEST_analysis\", experiment_params=experiment.params, k_to_calculate=(1,)\n",
+    ")\n",
+    "all_dependencies: Dict[Params, Data] = {\n",
+    "    pyscf_params: pyscf_hamiltonian,\n",
+    "    pp_params: trial_wf,\n",
+    "    blueprint_params: blueprint,\n",
+    "    simulated_experiment_params: experiment,\n",
+    "}\n",
+    "analysis = OverlapAnalysisData.build_analysis_from_dependencies(\n",
+    "    analysis_params, dependencies=all_dependencies\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally we save the wavefunction in a format that is acceptable for ipie."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.qcqmc.convert_to_ipie import save_wavefunction_for_ipie\n",
+    "ipie_data = save_wavefunction_for_ipie(\n",
+    "    hamiltonian_data=pyscf_hamiltonian, trial_wf_data=trial_wf, overlap_analysis_data=analysis, do_print=False\n",
+    ")\n",
+    "print(f\"Reconstructed shadow wavefunction energy: {ipie_data.variational_energy}\")\n",
+    "print(f\"Ideal trial wavefunction energy: {trial_wf.ansatz_energy}\")\n",
+    "print(f\"FCI energy: {trial_wf.fci_energy}\")\n",
+    "print(ipie_data)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run AFQMC \n",
+    "\n",
+    "Now that we have built our quantum wavefunction we can use it as a trial wavefunction in an AFQMC simulation. First we need to build a factorized Hamiltonian which is required for AFQMC. \n",
```

Comment:
Comma after wavefunction.  I think there should be one after First too.

Reply:
Done.